src/manage/keys/options.rs

```diff
@@ -26,13 +26,11 @@ enum Expiration {
 }
 
 /// Builds an [`Options`] object using [the Builder pattern][builder].
-///
-/// [builder]: https://rust-unofficial.github.io/patterns/patterns/creational/builder.html
```

Comment:
You need this line to provide the link above, here and elsewhere.  The docs job is failing in CI without it.